docs/reference/mapping/fields/source-field.asciidoc

```diff
@@ -6,6 +6,15 @@ at index time. The `_source` field itself is not indexed (and thus is not
 searchable), but it is stored so that it can be returned when executing
 _fetch_ requests, like <<docs-get,get>> or <<search-search,search>>.
 
+ifeval::["{release-state}"=="unreleased"]
+In some cases, particularly where reducing the volume of your stored data is a
+priority, you may consider either using <<synthetic-source,synthetic `_source`>>
+or <<disable-source-field,disabling the `_source` field>> altogether.
```

Comment:
Is it worth quickly describing the difference between these two options, or people need to go to their corresponding pages to find out?